certificate-policy.md

```diff
@@ -614,16 +638,15 @@ The Issuing CA shall revoke a Subordinate CA Certificate within seven (7) days i
 8. The Issuing CA's or Subordinate CA's right to issue Certificates under these Requirements expires or is revoked or terminated, unless the Issuing CA has made arrangements to continue maintaining the CRL/OCSP Repository;
 9. Revocation is required by this CP and/or the Issuing CA's CPS;
 10. The technical content or format of the Certificate presents an unacceptable risk to Application Software Suppliers or Relying Parties (e.g. the FPKIPA or CAB Forum might determine that a deprecated cryptographic/signature algorithm or key size presents an unacceptable risk and that such Certificates should be revoked and replaced by CAs within a given period of time); or
-11. The CA received a lawful and binding order from a government, judicial or regulatory body to
-revoke the Certificate.
+11. The CA received a lawful and binding order from a government, judicial or regulatory body to revoke the Certificate.
 
 ### 4.9.2 Who can request revocation
-The Subscriber, RA, or Issuing CA can initiate revocation of certificates.  The FPKIPA may also direct any revocation of a CA certificate.
+The Subscriber, RA, or CA can initiate revocation of certificates.  The FPKIPA may also direct any revocation of a CA certificate.
```

Comment:
Can the FPKIPA direct revocation of an end-entity certificate? May need clarifying.